Api/Services/PostingApiService.cs

```diff
@@ -85,10 +90,41 @@ public async Task<PostingResult> Apply(LeverSettings settings, LeverPostingApply
                 var client = _clientFactory.CreateClient();
                 var response = await client.PostAsync($"{URL}{settings.Site}/{model.PostingId}?key={settings.ApiKey}", model.ToFormData());
 
-                return JsonConvert.DeserializeObject<PostingResult>(await response.Content.ReadAsStringAsync());
+                var json = JsonConvert.DeserializeObject<PostingResult>(await response.Content.ReadAsStringAsync());
+
+                if (json.Ok)
+                {
+                    await _workflowManager.TriggerEventAsync(
```

Comment:
I'd suggest adding a method for the trigger of an event as the only differences between line 97 and 115 is the value of `IsSuccessful` (e.g. `TriggerNotificationEvent(false)` where false represents the value of `IsSuccessful`.